### PR TITLE
General Momentum FGD cleanup

### PR DESCRIPTION
**Pull request by tmob03**
Definitely not exhaustive, was doing a quick glance through entities. Removes lots of noticed dysfunction hl2 entities, some vgui entities and a few multiplayer entities.

### CODE REVIEW

File: fgd/point/env/env_outtro_stats.fgd [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@PointClass base(BaseEntityPoint) 
+@PointClass appliesto(+USE_MULTIPLAYER)
```
Comment (braem) [inline fgd/point/env/env_outtro_stats.fgd:1]: I don't believe this is a multiplayer entity

File: db/factories/momentum.txt [hunk 2]
```diff
@@ -382,10 +381,6 @@ trigger_tractorbeam
 trigger_userinput
 trigger_vphysics_motion
 trigger_wind
-vgui_indicator_panel
-vgui_movie_display
-vgui_screen
-vgui_slideshow_display
```
Comment (braem) [inline db/factories/momentum.txt:388]: begrudgingly, I don't believe now is the time to remove VGUI entities - we don't have good alternatives at the moment.

**Review by braem — CHANGES_REQUESTED** [review]

File: fgd/point/env/env_outtro_stats.fgd (hunk 1 above)
Comment (tmob03) [reply fgd/point/env/env_outtro_stats.fgd:1]: > I don't believe this is a multiplayer entity

It's from l4d, it controls the end of campaign stats, so I guess you're right. Would I give -MOMENTUM?

Comment (braem) [reply fgd/point/env/env_outtro_stats.fgd:1]: Good to remove this entirely. We'll rip it out of game code because it is really l4d specific.

Comment (tmob03) [reply fgd/point/env/env_outtro_stats.fgd:1]: Alright

File: db/factories/momentum.txt [hunk 3]
```diff
@@ -385,7 +384,7 @@ trigger_wind
 vgui_indicator_panel
 vgui_movie_display
 vgui_screen
-vgui_slideshow_display
+vgui_slideshow_displa
```
Comment (braem) [inline db/factories/momentum.txt:387]: typo on readding

File: fgd/point/env/env_outtro_stats.fgd [hunk 4]
```diff
@@ -1,12 +0,0 @@
-@PointClass base(BaseEntityPoint) 
-	size(-8 -8 -8, 8 8 8)
-= env_outtro_stats: "An entity to control the outtro stats at the end of a campaign."
```
Comment (braem) [inline fgd/point/env/env_outtro_stats.fgd:3]: Should also remove from p2ce's factory and visgroups.cfg

**Review by braem — CHANGES_REQUESTED** [review]

File: db/factories/momentum.txt (hunk 3 above)
Comment (tmob03) [reply db/factories/momentum.txt:387]: Oops, sorry

**Review by braem — APPROVED** [review]

**Pull request merged**